### ISSUE TITLE
[Moved from Discord] Smoke bomb delay between animation and active effect

### ISSUE THREAD

**Issue by WhitemaneSylah** [gamefreedomgit/Maelstrom#2257]
ivarx
OP
 — Today at 11:55
Description: :
Smoke bomb effect is delayed a lot: Any spell from out side smoke bomb can reach the target within the smoke bomb
one second or more after the effect has already finished.

How to reproduce it::
As you can see in the two videos below the warrior is able to charge a rogue even 
if smoke bomb animation has already finished.
You can also noticed in the left of my character, there is an addon that process the combat log and shows the smoke bomb aura.
Once this addon manage to show it this mean that the aura is already in place (because of the processing time required).
This issue is not related to any lag because i am playing with 15 ms (i live in france near to OVH).

If needed, you can replay the two video below with *0.25 speed to see exactly what's happening

https://www.twitch.tv/videos/1750617976
https://www.twitch.tv/videos/1750617974

How it should work::
Creates a cloud of thick smoke in an 8 yard radius around the Rogue for 5sec. Enemies are unable to target into or out of the smoke cloud.
The effect of the smoke bomb should be applied once the aura animation is done and should not be delayed 

TLDR - Upon cast of Smoke bomb their should be no delay between the animation start and the debuff being applied.
